Dockerfile

```diff
@@ -31,15 +34,21 @@ RUN python -m coverage run tests/test_${package}.py \
 # STAGE 2 -- Build final plugin image
 FROM quay.io/arcalot/arcaflow-plugin-baseimage-python-osbase:0.5.0
 ARG package
-RUN dnf -y install systemd \
- && dnf -y install pcp pcp-export-pcp2json pcp-system-tools procps-ng util-linux-core
+RUN dnf -y install systemd pcp pcp-export-pcp2json pcp-system-tools procps-ng util-linux-core \
+ && useradd -U pcp
```

Comment:
Without the `systemd` install, [the PCP installation produces repeated warnings](https://github.com/arcalot/arcaflow-plugin-pcp/actions/runs/16748784747/job/47413560836?pr=54#step:8:353):
```
warning: user pcp does not exist - using root
warning: group pcp does not exist - using root
```
@dustinblack, are you sure you want to add the `pcp` user _after_ the `pcp` installation?  (If so, an explanatory comment might be a good idea.)